tests/cunit/CMakeLists.txt

```diff
@@ -68,6 +68,8 @@ if (NOT PIO_USE_MPISERIAL)
   target_link_libraries (test_pioc_fill pioc)
   add_executable (test_darray EXCLUDE_FROM_ALL test_darray.c test_common.c)
   target_link_libraries (test_darray pioc)
+  add_executable (test_darray_uneven EXCLUDE_FROM_ALL test_darray_uneven.c test_common.c)
```

Comment:
This test does not exist in the code. If you created it, try git add test_darray_uneven.c. If not, then delete all references to it in cmake.

Reply:
oops - I thought that I had added it. I will correct - thanks.